src/lib/openapi/spec/addon-schema.test.ts

```diff
@@ -5,6 +5,8 @@ test('addonSchema', () => {
     const data: AddonSchema = {
         provider: 'some-provider',
         enabled: true,
+        description: null,
+        id: 5,
```

Comment:
These fields are now mandatory when when returning the schema. In effect they were there before, but the schema didn't mark them as mandatory because it was also used for creating addons.

Reply:
Same goes here, I like that we're doing the work with separating request/response to avoid having optional fields, or having to allow extra properties in responses because we're reusing the schema as a request